ts/Dashboards/SerializeHelper/JSONConnectorHelper.ts

```diff
@@ -0,0 +1,131 @@
+/* *
+ *
+ *  (c) 2009 - 2023 Highsoft AS
+ *
+ *  License: www.highcharts.com/license
+ *
+ *  !!!!!!! SOURCE GETS TRANSPILED BY TYPESCRIPT. EDIT TS FILE ONLY. !!!!!!!
+ *
+ *  Authors:
+ *  - Pawel Lysy
+ *
+ * */
+
+'use strict';
+
+/* *
+ *
+ *  Imports
+ *
+ * */
+
+import type Globals from '../Globals';
+import type JSON from '../JSON';
+import type JSONConnectorOptions from '../../Data/Connectors/JSONConnectorOptions';
+
+import JSONConnector from '../../Data/Connectors/JSONConnector.js';
+import DataTableHelper from './DataTableHelper.js';
+import Serializable from '../Serializable.js';
+import U from '../../Core/Utilities.js';
+const { merge } = U;
+
+/* *
+ *
+ *  Functions
+ *
+ * */
+
+/**
+ * Converts the given JSON to a class instance.
+ *
+ * @param {JSONConnectorHelper.JSON} json
+ * JSON to deserialize as a class instance or object.
+ *
+ * @return {JSONConnector}
+ * Returns the class instance or object, or throws an exception.
+ */
+function fromJSON(
+    json: JSONConnectorHelper.JSON
+): JSONConnector {
+    return new JSONConnector(json.options);
+}
+
+/**
+ * Validates the given class instance for JSON support.
+ *
+ * @param {Globals.AnyRecord} obj
+ * Class instance or object to validate.
+ *
+ * @return {boolean}
+ * Returns true, if the function set can convert the given object, otherwise
+ * false.
+ */
+function jsonSupportFor(obj: Globals.AnyRecord): obj is JSONConnector {
```

Comment:
In some places you use `json` in other `JSON`, would be good to unify it everywhere.

Reply:
It is coherent with the other Helper classes, so I guess, that it could stay like that.